### DISCUSSION TITLE
编译问题，/bin/sh: --cpp_out: command not found

### DISCUSSION THREAD

**Discussion by wangxucumt** [apache/doris#6953] [RFC]
[ 98%] Built target rpc_replay
[ 98%] Running C++ protocol buffer compiler on view.proto
/bin/sh: --cpp_out: command not found
make[2]: [tools/rpc_view/view.pb.cc] Error 127
make[1]: [tools/rpc_view/CMakeFiles/rpc_view.dir/all] Error 2
编译DORIS的时候，报错了，GCC使用的是7.5版本，有同学遇到过这个问题嘛，是否跟GCC版本有关系

**Comment by hf200012** [comment]
Please provide the docker image version and doris version you want

**Reply by wangxucumt** [reply]
doris的版本就是从master分支拉取的，没有在docker下编译，直接在centos7.8环境下编译，GCC 10+, Oracle JDK 1.8+, Python 2.7+, Apache Maven 3.5+, CMake 3.19.2+ Bison 3.0+相关版本都满足，开始以为GCC版本问题，后来升级到了10.2版本，还是同样的错误

**Comment by dataalive** [comment]
using the latest ldb toolchain to build from source will be more easily
